tests/_test_util.py

```diff
@@ -0,0 +1,118 @@
+from string import ascii_letters
+from typing import Collection, Tuple
+
+import numpy as np
+import pandas as pd
+from anndata import AnnData
+from scipy import sparse
+
+
+def gen_adata(
```

Comment:
In general: I like this function a lot, but I think we have to alter it a bit (does not have to be this PR), since it does not displays our data well (like multiple events of one patient for example).

Reply:
Perfectly agree. It was just copied from AnnData :)